.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["es2015", "stage-0"]
```

Comment:
Have you checked out the new babel preset env. Babel says basically all others are now deprecated

Reply:
i will do, will change it!